travis.sh

```diff
@@ -51,6 +51,7 @@ if ! [ "$IN_DOCKER" ]; then
         -e TEST_BLACKLIST \
         -v $(pwd):/root/$REPOSITORY_NAME \
         -v $HOME/.ccache:/root/.ccache \
+        -t \
```

Comment:
i don't think this belongs here

Reply:
Can you elaborate on this? Why do you think so?
This was a cherry-pick from https://github.com/ros-industrial/industrial_ci/pull/128 as stated in the commit comment.

Reply:
@davetcoleman Ping

Reply:
I don't normally look at individual commit comments, and I know future users won't either. Can you add this as a comment in the travis file itself?

> run docker tests with tty to avoid warnings about unknown terminal size